src/domain/validations.ts

```diff
@@ -203,7 +223,12 @@ function countWords(str: string) {
   return str.split(' ').length;
 }
 
-function validatePersonalInfo(info: PersonalInfo, errors: SectionError[], validateGoogleEmailRequired: boolean = true) {
+function validatePersonalInfo(
+  info: PersonalInfo,
+  errors: SectionError[],
+  validateGoogleEmailRequired: boolean = true,
+  validateWebsiteRequired: boolean = false,
```

Comment:
On deeper thought, in the future we can change this third argument to be an options object, which allows for description, and not having to pass a bunch of `false` arguments if we want to pass a true at an nth argument.

It would also keep the arity low.